### ISSUE TITLE
FOIL Request Page Accessibility

### ISSUE THREAD

**Issue by joelbcastillo** [CityOfNewYork/NYCOpenRecords#306]
As a user who is visually impaired, I need to be able to use a screen reader on the request page so that I can view the documents / responses that have been posted.

Acceptance Criteria:
- Responses are shown in a table made of divs. Aria tags should be used to describe the table to allow the screen reader to navigate by cell.
- Ensure the hierarchy of elements and formatting is consistent (H1 -> P)
- Ensure that all clickable elements are described properly.

An example of using aria to make a table accessible is available on Stack Overflow - https://stackoverflow.com/questions/31008192/using-aria-to-make-a-div-table-accessible

Some resources about using ARIA to make forms accessible.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/forms/Basic_form_hints#Labeling_with_ARIA
